platform-sdk/docs/hashgraph_glossary.md

```diff
@@ -0,0 +1,250 @@
+# Hashgraph Glossary
+
+<a name="top"/>
+
+- [Rounds](#rounds)
+- [Fields of an event](#fields-of-an-event)
+- [Events](#events)
+- [Event relationships](#event-relationships)
+- [Generations](#generations)
+- [Address Books](#addressbooks)
+- [Parameters](#parameters)
+
+<a name="rounds"/>
+
+## Rounds
+
+- __Pending__ round - the round number of the earliest round that hasn't yet reached consensus. This is the round whose
+  judges are currently being determined. Those consensus calculations use the roster whose roster round is the current
+  `pendingRound`. Any new event is created containing a `birthRound` field equal to the current `pendingRound`. All
+  events currently in the hashgraph should have birth rounds in the range from `pendingRound - roundsAncient` to
+  `pendingRound`, inclusive.
+- __Consensus__ round - the round number for the latest round that has reached consensus. <br>
+    - `consensusRound = pendingRound - 1`
+- __First non-ancient__ round - the oldest round that is not ancient<br>
+    - `firstNonAncientRound = pendingRound - roundsAncient`
+- __First non-expired__ round - the oldest round that is not expired<br>
+    - `firstNonExpiredRound = pendingRound - roundsExpired`
+- __Last roster__ round - Last future round for which there should be a roster in the queue<br>
+    - `lastRosterRound = pendingRound + roundsRoster`
```

Comment:
The terms `first` and `last` are a bit unnatural to me when describing the hashgraph which we think of as growing upward. I can get used to them, but the terms `earliest` and `latest` are more familiar.

Reply:
I agree